package.json

```diff
@@ -25,8 +25,12 @@
   "license": "MIT",
   "scripts": {
     "android:dev-client": "IS_DEV=1 npx expo run:android",
-    "bootstrap:new_app": "yarn && node ./scripts/bootstrap.js",
-    "build:production:android": "yarn prepare:production && eas build --platform android --profile production",
+    "baca:bootstrap": "yarn build:baca-cli && node ./scripts/cli/build bootstrap",
+    "baca:generate": "yarn build:baca-cli && node ./scripts/cli/build generate",
+    "b": "yarn build:baca-cli && node ./scripts/cli/build b && yarn eslint src --fix && yarn tsc",
```

Comment:
Should there be a differance between `baca:bootstrap` <-> `b` and `baca:generate` <-> `g` 🤔